src/class-parsely.php

```diff
@@ -1119,13 +1137,32 @@ public function insert_parsely_javascript() {
 		 * If true, the JavaScript files are sourced.
 		 *
 		 * @since 2.2.0
+		 * @deprecated deprecated since version 2.5.0
 		 *
 		 * @param bool $display True if the JavaScript file should be included. False if not.
 		 */
-		if ( ! apply_filters( 'parsely_filter_insert_javascript', $display ) ) {
+		if ( ! apply_filters_deprecated(
+			'parsely_filter_insert_javascript',
+			array( $display ),
+			'2.5.0',
+			'wp_parsely_insert_javascript'
+		) ) {
 			return;
 		}
 
+		/**
+		 * Filters whether to include the Parsely JavaScript file.
+		 *
+		 * If true, the JavaScript files are sourced.
```

Comment:
Since the conditions aren't about requiring a JS file or not, but more about whether some JS files are enqueued or not, can this description be updated?

Is "insert javascript" still the right phrase to convey what really happens after this point? Maybe something like "Insert tracker", "Apply tracking", "Track response" or similar? Yes, we're registering and enqueueing JS, but what feature is it that we're actually manipulating here from a user's perspective?

Reply:
#294 is following this up.